anitya/__init__.py

```diff
@@ -145,7 +145,8 @@ def log(session, project=None, distro=None, topic=None, message=None):
                              'project: %(project)s',
         'project.edit': '%(agent)s edited the project: %(project)s fields: '
                         '%(changes)s',
-
+        'project.flag': '%(agent)s flagged the project: %(project)s',
```

Comment:
Yes I think it would be useful.

As it is for admins only, we could still easily rework that later on :)